### DISCUSSION TITLE
BurmillaOS on DistroWatch?

### DISCUSSION THREAD

**Discussion by gramian** [burmilla/os#167] [Ideas]
Dear Community,

checking on [DistroWatch](https://distrowatch.com/), I was unable to find BurmillaOS, while [RancherOS](https://distrowatch.com/table.php?distribution=rancheros) is listed. Is it planned to be [submitted](https://distrowatch.com/dwres.php?resource=submit) in the future or are there reservation against this Linux distribution index?

Thank you for explaining

**Comment by olljanat** [comment]
Don't use DistroWatch so haven't even think about it earlier.

However because lack of maintainers #2 and only very small number of contributors it is not too sure what is future of this project so I prefer to be careful with promises we give.

So if BurmillaOS gets added to their list it most probably should be with status "Dormant" (look [definitions](https://distrowatch.com/dwres-mobile.php?resource=faq#distrostatus) ) as 2.0.0 most likely will be last release version for long time and after that only new Docker versions will be added to https://github.com/burmilla/os-services/blob/master/index.yml

That can be done because kernel 5.10.x is supported [until December 2026](https://kernel.org/category/releases.html) and we only need create new release when there is security or other critical issues fixed on those.
